client/aws/errors.go

```diff
@@ -11,8 +11,23 @@ const (
 	ELBConfigurationMismatch = "already exists and it is configured with different parameters"
 )
 
-var (
-	malformedAmazonAccountIDError   = errgo.New("amazon account ID can only contain numbers")
-	wrongAmazonAccountIDLengthError = errgo.New("amazon account ID has the wrong size")
-	emptyAmazonAccountIDError       = errgo.New("amazon account ID cannot be empty")
-)
+var malformedAmazonAccountIDError = errgo.New("amazon account ID can only contain numbers")
```

Comment:
The error messages by convention align to the error variable name. The error message here should be `malformed amazon account id`. More details for the specific use case should be added vie `microerror.MaskAnyf`.